shell/scl_source

```diff
@@ -47,8 +47,7 @@ for arg in "$@"; do
     done
 
     # Now check if the collection isn't already enabled
-    /usr/bin/scl_enabled $arg > /dev/null 2> /dev/null
-    if [ $? -ne 0 ]; then
+    if !/usr/bin/scl_enabled $arg > /dev/null 2> /dev/null; then
```

Comment:
There's a syntax error - a space is missing between the `!` and the command.

Reply:
Gah! Thanks! (/me was too lazy and figured I could get away with GitHub web interfaces for a one liner, had the space in the version I had tested)